app/views/layouts/application.html.erb

```diff
@@ -6,11 +6,23 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```

Comment:
Great job adding your cdn assets to a partial to clean up the application layout file!